### PR TITLE
Homework 3 (Password Check using any_of)

### PR DESCRIPTION
**Pull request by McKucia**
Another solution

### CODE REVIEW

File: module3/homework/passwordCheck/validation.cpp [hunk 1]
```diff
@@ -1,2 +1,67 @@
+#include <iostream>
+#include <array>
+#include <algorithm>
```
Comment (danielzwierz) [inline module3/homework/passwordCheck/validation.cpp:3]: Sort

File: module3/homework/passwordCheck/validation.cpp [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+
+std::string getErrorMessage(ErrorCode errorType){
```
Comment (danielzwierz) [inline module3/homework/passwordCheck/validation.cpp:7]: space

File: module3/homework/passwordCheck/validation.cpp [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+    switch(errorType){
```
Comment (danielzwierz) [inline module3/homework/passwordCheck/validation.cpp:8]: space

File: module3/homework/passwordCheck/validation.cpp [hunk 4]
```diff
@@ -1,2 +1,67 @@
...
+        case ErrorCode::PasswordsDoesNotMatch:
+            return "Passwords does not match. \n";
+            break;
+        default:
+            return "Unknown error \n";
+    }
+}
+
+bool doesPasswordsMatch(const std::string& first, const std::string& second)
+{
```
Comment (danielzwierz) [inline module3/homework/passwordCheck/validation.cpp:33]: use consistant code format, always put curly brace in the same line

File: module3/homework/passwordCheck/validation.hpp [hunk 5]
```diff
@@ -1,2 +1,18 @@
-// TODO: I'm empty :) Put enum and function headers here.
-// Don't forget the header guard
\ No newline at end of file
+#include <string>
```
Comment (m-kop) [inline module3/homework/passwordCheck/validation.hpp:1]: header guard missing

File: module3/homework/passwordCheck/validation.cpp [hunk 6]
```diff
@@ -1,2 +1,67 @@
...
+#include <algorithm>
+
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+
+std::string getErrorMessage(ErrorCode errorType){
+    switch(errorType){
+        case ErrorCode::Ok:
+            return "OK";
```
Comment (m-kop) [inline module3/homework/passwordCheck/validation.cpp:10]: missing "\n", you should be consistent with the other case's

File: module3/homework/passwordCheck/validation.cpp [hunk 7]
```diff
@@ -1,2 +1,67 @@
+#include <iostream>
+#include <array>
+#include <algorithm>
+
 #include "validation.hpp"
```
Comment (m-kop) [inline module3/homework/passwordCheck/validation.cpp:5]: sort

File: module3/homework/passwordCheck/validation.hpp (hunk 5 above)
Comment (danielzwierz) [inline module3/homework/passwordCheck/validation.hpp:1]: header guard

File: module3/homework/passwordCheck/validation.cpp (hunk 3 above)
Comment (McKucia) [reply module3/homework/passwordCheck/validation.cpp:8]: where?

File: module3/homework/passwordCheck/validation.cpp (hunk 6 above)
Comment (McKucia) [reply module3/homework/passwordCheck/validation.cpp:10]: ye i know, but tests expect only "OK"

**Pull request closed without merging**